src/components/pill-selector-dropdown/PillSelectorDropdown.stories.js

```diff
@@ -202,6 +203,71 @@ export const showRoundedPills = () => {
     );
 };
 
+export const roundPillVariants = () => {
+    const storeWithPills = new Store({
+        error: '',
+        selectedOptions: [
+            {
+                displayText: 'default@example.com',
+                value: '1',
+            },
+            {
+                displayText: 'warning@example.com',
+                value: '2',
+            },
+            {
+                displayText: 'waived@example.com',
+                value: '3',
+            },
+            {
+                displayText: 'waived+external@example.com',
+                value: '4',
+                isExternalUser: true,
+            },
+        ],
+        selectorOptions: [],
+    });
+    const getPillVariant = ({ value }) => {
+        switch (value) {
+            case '2':
+                return PILL_VARIANT_WARNING;
+            case '3':
+            case '4':
+                return PILL_VARIANT_WAIVED;
+            case '1':
+            default:
+                return PILL_VARIANT_DEFAULT;
+        }
+    };
+    const { handleInput, handleRemove, handleSelect, validator, validateForError } = generateProps(storeWithPills);
+    return (
+        <State store={storeWithPills}>
+            {state => (
+                <PillSelectorDropdown
+                    allowCustomPills
+                    error={state.error}
+                    getPillVariant={getPillVariant}
+                    placeholder="Names or email addresses"
+                    onInput={handleInput}
+                    onRemove={handleRemove}
+                    onSelect={handleSelect}
+                    selectedOptions={state.selectedOptions}
+                    selectorOptions={state.selectorOptions}
+                    showRoundedPills
```

Comment:
I was looking into breaking this into two stories but noticed that both of the current ones show the avatar. I think it's because `showAvatars` is not passed to the pill. Do you know if this is expected? 

https://github.com/box/box-ui-elements/blob/master/src/components/pill-selector-dropdown/PillSelector.js#L225

I can probably fix this if this was not intentional.

Reply:
i dont think it was intentional but it might be better to address it outside this ticket since it would impact a lot of modals that are using the RoundPill. feel free to leave it for now and file a bug thanks!